model/contract/UsageControlEnpoint.ttl

```diff
@@ -0,0 +1,59 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix ids: <https://w3id.org/idsa/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Classes
+# -------
+
+ids:UsageControlEndpoint
+    a owl:Class;
+    rdfs:label "Usage policy class"@en ;
+    rdfs:comment "Defined usage control endpoint in the IDS."@en;
```

Comment:
We would suggest the following changes:

ids:UsageControlComponentEndpoint
    a owl:Class;
    rdfs:label "Usage Control Component Endpoint class"@en ;
    rdfs:comment "Defined usage control component endpoints (i.e. PIP, PXP, etc.) in the IDS."@en;